docs/tutorials/migrate-to-chiselled-rock.rst

```diff
@@ -5,7 +5,8 @@ Prerequisites
 -------------
 - snap enabled system (https://snapcraft.io)
 - LXD installed (https://linuxcontainers.org/lxd/getting-started-cli/)
-- skopeo installed (https://github.com/containers/skopeo)
+- skopeo installed (https://github.com/containers/skopeo).
+  A Skopeo snap will also be automatically installed as a Rockcraft dependency
```

Comment:
It's not a skopeo snap, skopeo is built as a binary inside the rockcraft snap. 
But I don't know how relevant that difference is (maybe I'm being too picky), how about we just remove this reference to skopeo in the Prerequisites section instead?